src/components/collapsible-panel.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { caretDownIcon, caretRightIcon } from '@jupyterlab/ui-components';
+
+export interface ICollapsiblePanelProps {
+  title: JSX.Element | string;
+  content: JSX.Element | string;
```

Comment:
Does it make sense for us to use `children` for this? Maybe review how other UI libraries handle this.

Reply:
Material Accordion [uses](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Accordion/Accordion.js#L151) `children` for both Title/Summary and Content where first element of `children` is Title/Summary and everything else is Content by convention. I think it's a straightforward approach (going to implement it now). But [based on other comment](https://github.com/jupyter-server/jupyter-scheduler/pull/12#discussion_r974614651), Title should still be a prop, right?

Reply:
Component is now using `children` for Content and separate `props.title` for Title (with `string` type enforced)